src/miniscript/context.rs

```diff
@@ -567,8 +589,11 @@ impl ScriptContext for Segwitv0 {
         ms.ext.max_sat_size.map(|x| x.0)
     }
 
-    fn pk_len<Pk: MiniscriptKey>(_pk: &Pk) -> usize {
-        34
+    fn pk_len<Pk: MiniscriptKey>(pk: &KeyExpr<Pk>) -> usize {
+        match pk {
+            KeyExpr::<Pk>::SingleKey(_) => 34,
+            KeyExpr::<Pk>::MuSig(_) => 32,
```

Comment:
Here and everywhere, where we `32`. It should actually be `33` as it also includes the push len prefix.

Reply:
Done. changed everywhere.